ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -6,11 +6,20 @@ import {
   handleSettingsRefs,
 } from '../../../helpers/utils/settings-search';
 import { MetaMetricsEventCategory } from '../../../../shared/constants/metametrics';
+///: BEGIN:ONLY_INCLUDE_IN(build-main,build-mmi,build-beta)
+import Typography from '../../../components/ui/typography/typography';
+///: END:ONLY_INCLUDE_IN
 import { Text } from '../../../components/component-library';
 import {
-  FontWeight,
+  ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-mmi,build-beta)
+  FONT_WEIGHT,
```

Comment:
Should be `FontWeight` instead of `FONT_WEIGHT`... this is causing the lint to fail